hbw/app/assets/stylesheets/hbw.sass

```diff
@@ -491,3 +489,10 @@ input.hbw-checkbox
   &[type=checkbox]
     + span:before
       content: "\f0c8"
+
+.react-select__control--is-disabled
```

Comment:
I believe it works but have other way to customize react-select
https://github.com/latera/homs/blob/f680740201b65dccd09ae071edfb3e5501a9c4b7/hbw/app/javascript/packs/hbw/components/form/select.js.jsx#L125